lib/codegen/meta/isa/x86/recipes.py

```diff
@@ -544,6 +544,16 @@ def floatccs(iform):
         sink.put8(imm as u64);
         ''')
 
+# XX /n Unary with floating point immediate.
+# FIXUP: ^ I am unusure about the notation above.
```

Comment:
The notation above is right, but the comment should be more specific, because this recipe will only be usable for 0.0 and not other floating-point immediates.